packages/cadl-ranch-specs/http/enums/fixed/main.cadl

```diff
@@ -0,0 +1,48 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Azure.Core;
+using Cadl.Http;
+
+@scenarioService("/fixed-enums")
+namespace FixedEnums;
```

Comment:
```suggestion
@scenarioService("/enums/fixed")
namespace Enums.Fixed;
```

So we can get this structure in the ui
Enums:
   Fixed:
      ...
   Extensible:
      ...